src/other/Table.js

```diff
@@ -1,12 +1,13 @@
 "use strict";
 (function (root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "../common/HTMLWidget", "./Paginator", "../common/Utility", "css!./Table"], factory);
+        define(["d3", "../common/HTMLWidget", "./Paginator", "../common/Utility", "../common/Widget", "css!./Table"], factory);
```

Comment:
This is the first issue, I realize this cannot be the right way to do this, requiring HTMLWidget and Widget at the same time since HTMLWidget extends Widget.   But in order to accomplish this task I need to be able to test if a data item is widget, obviously, for example here https://github.com/hpcc-systems/Visualization/pull/1237/files#diff-164179c53aa545afdc2035238236796aR345.  What would be the appropriate way to accomplish that?